roles/ocp/templates/install-config.yaml.j2

```diff
@@ -48,6 +48,7 @@ platform:
     externalNetwork: {{ item.cloud.external_network }}
     apiFloatingIP: {{ item.cloud.api_floating_ip }}
     ingressFloatingIP: {{ item.cloud.ingress_floating_ip }}
+    profile: rhacm-automation
```

Comment:
It should not be set here.
Instead the value should be changed in line 47.
Also, the generation flow of "clouds.yaml" should be changed as well.